src/tree/browser/tree.tsx

```diff
@@ -0,0 +1,501 @@
+/********************************************************************************
```

Comment:
Could consider in future to split this file up. Quite large at the moment.
Could be for example separate renderers.

Reply:
https://github.com/eclipse-cdt-cloud/vscode-peripheral-inspector/pull/46 already moves the renderer(s) outside. Taking the changes over should improve it.

Reply:
Great, let's do this then once that the Peripheral Inspector PR is merged.